samples/MobileBuyIntegration/app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,20 @@
     <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
     <uses-permission android:name="android.permission.READ_MEDIA_VISUAL_USER_SELECTED" />
 
+    <queries>
+        <!-- for supporting deep links to the maps app -->
+        <intent>
+            <action android:name="android.intent.action.VIEW" />
+            <data android:scheme="geo" />
+        </intent>
+
+        <!-- for supporting deep links to the messaging app -->
+        <intent>
+            <action android:name="android.intent.action.SENDTO" />
+            <data android:scheme="sms" />
+        </intent>
+    </queries>
```

Comment:
It looks like behaviour depends on API version.

For < 11:

It sounds a bit like a free-for-all, let me do some testing...

For >= 11:

When querying for info about other apps, the system filters info. So the code wouldn't detect that there's anything able to handle the scheme, and we'd log a warning instead of launching the intent.

And the `<queries>` element declares which apps you want to interact with and makes them visible. Inside queries you could also make it a specific package if you want a specific app instead of any app that can handle the intent as shown above. So, it should give quite good flexibility for clients to decide what they want to launch.

(https://developer.android.com/training/package-visibility)

Reply:
I tested pre 11, yeah it opens up e.g. the maps, and messaging app even if the queries element is commented/missing from the manifest.

👍 I added a little more info to the README